c1-chat/src/app/globals.css

```diff
@@ -0,0 +1,29 @@
+@import "tailwindcss";
+
```

Comment:
i think we can clean this file up. none of these variables will get used anyways